qa/L0_model_update/instance_update_test.py

```diff
@@ -311,10 +311,10 @@ def test_infer_while_updating(self):
         # Unload model
         self.__unload_model()
 
-    # Test instance resource requirement increase
+    # Test instance resource requirement update
     @unittest.skipUnless(os.environ["RATE_LIMIT_MODE"] == "execution_count",
                          "Rate limiter precondition not met for this test")
-    def test_instance_resource_increase(self):
+    def test_instance_resource_update(self):
         # Load model
```

Comment:
Can you describe the steps in the sub-test?

Reply:
Yes. I will update as I update the tests on https://github.com/triton-inference-server/server/pull/5885#discussion_r1213864265